nccr-mediality.csl

```diff
@@ -311,6 +311,7 @@
         <else-if position="subsequent">
           <text macro="contributors-short" suffix=", "/>
           <text macro="title-short"/>
+          <text variable="first-reference-note-number" prefix=" (Anm. " suffix=")"/>
```

Comment:
@cnotz, just curious, but what does "Anm" stand for here?

@adam3smith, do you know if we have a CSL term for this?